src/infra/src/db/postgres.rs

```diff
@@ -293,7 +295,7 @@ impl super::Db for PostgresDb {
         if need_watch {
             let cluster_coordinator = super::get_coordinator().await;
             cluster_coordinator
-                .put(key, Bytes::from(""), true, start_dt)
+                .put(key, Bytes::from(need_watch_dt.to_string()), true, start_dt)
```

Comment:
Here, maybe we can set the third parameter `start_dt` instead of set it into body.